requirements.yml

```diff
@@ -23,3 +23,7 @@ collections:
     type: git
   - name: https://github.com/ansible-collections/community.libvirt
     type: git
+  - name: https://github.com/ansible-collections/kubernetes.core
```

Comment:
May we need to add this deps in ansible-requirements.txt?
```
kubernetes >= 12.0.0
PyYAML >= 3.11
```

Reply:
One version of kubernetes sure.

The other might be transitive and also default in most host.

Reply:
It misses it, should I make the playbook without kubernetes.core module ?

Reply:
python3-kubernetes exists in epel,
Should I make the change as 2 commit ?
One with kubernetes.core expecting preinstalled, 2th is just with oc expecting the login session already there ?

Reply:
Note: these dependencies will be covered in #196 - it should merge shortly.

Reply:
https://review.rdoproject.org/zuul/build/22436e542b2b46698a9dfa6d22e142a7